docs/notebooks/image_to_image.ipynb

```diff
@@ -1,23 +1,10 @@
 {
```

Comment:
what happend here? the notebook changed but not the md file?

Reply:
ah, I didn't mean to commit that, i think since the notebook file records metadata such as number of runs, running the notebook causes changes that I committed mistakenly

Reply:
I attempted to revert the changes to that file to main but it has since been updated there as well, after merging there will be no difference to either the notebook or the markdown